package.json

```diff
@@ -15,7 +15,7 @@
     "@0x/sol-trace": "^2.0.4",
     "chokidar-cli": "^1.2.2",
     "dotenv": "^6.2.0",
-    "eslint": "^5.13.0",
+    "eslint": "5.12.0",
```

Comment:
What broke?

Reply:
package `react-scripts` requires (in the current version 2.1.8) version 5.12.0 of eslint. Master branch of 'react-scripts' has bumped it to 5.16.0 so this downgrade and pinning for our project is temporary.